### PR TITLE
fix(trace-viewer): trap focus inside drop-target popup

### PR DESCRIPTION
**Pull request by mxschmitt**
Trapping the focus seems to be common practise when dealing with in-page popups.

https://github.com/microsoft/playwright/issues/29099

### CODE REVIEW

File: packages/trace-viewer/src/ui/workbenchLoader.tsx [hunk 1]
```diff
@@ -159,17 +161,17 @@ export const WorkbenchLoader: React.FunctionComponent<{
         <div>3. Drop the trace from the download shelf into the page</div>
       </div>
     </div>}
-    {!isServer && !dragOver && !fileForLocalModeError && (!traceURLs.length || processingErrorMessage) && <div className='drop-target'>
+    {showFileUploadDropArea && <div className='drop-target'>
       <div className='processing-error' aria-live='assertive'>{processingErrorMessage}</div>
-      <div className='title' role='heading'>Drop Playwright Trace to load</div>
+      <div className='title' role='heading' aria-level={1}>Drop Playwright Trace to load</div>
```
Comment (mxschmitt) [inline packages/trace-viewer/src/ui/workbenchLoader.tsx:166]: drive-by: a11y fix

File: packages/trace-viewer/src/ui/workbenchLoader.tsx [hunk 2]
```diff
@@ -159,17 +161,17 @@ export const WorkbenchLoader: React.FunctionComponent<{
...
       <div>or</div>
       <button onClick={() => {
         const input = document.createElement('input');
         input.type = 'file';
         input.multiple = true;
         input.click();
         input.addEventListener('change', e => handleFileInputChange(e));
-      }}>Select file(s)</button>
+      }} type='button'>Select file(s)</button>
```
Comment (mxschmitt) [inline packages/trace-viewer/src/ui/workbenchLoader.tsx:174]: drive-by: a11y fix

File: packages/trace-viewer/src/ui/workbench.tsx [hunk 3]
```diff
@@ -213,7 +214,7 @@ export const Workbench: React.FunctionComponent<{
   else if (model && model.wallTime)
     time = Date.now() - model.wallTime;
 
-  return <div className='vbox workbench'>
+  return <div className='vbox workbench' {...(inert ? { inert: '' } : {})}>
```
Comment (dgozman) [inline packages/trace-viewer/src/ui/workbench.tsx:217]: I'd think `inert="true"` as a string is more readable.

**Review by dgozman — APPROVED** [review]
